docs/protocol.md

```diff
@@ -290,9 +290,7 @@ POST / HTTP/1.1
 ```json
 {
   "patchDataHash": "Hash of the patch data.",
-  "recoveryKey": {
-    "publicKeyHex": "A SECP256K1 public key expressed in compressed HEX format."
-  },
+  "recoveryKey": "A SECP256K1 public key expressed in compressed JWK format.",
```

Comment:
I thought we are abandoning `protocol.md`?

Reply:
yes, I also feel like we should not be updating that, and instead we should be updating the swagger yaml or the core spec...

Reply:
We talked about having a separate API spec, as opposed to having this definition in the main core spec. The swagger yaml does not contain the Core service API yet, I filed #537 for moving definitions to the yaml and would like to address it separately, meanwhile updated this doc so that the information is not out of date.